tinygrad/nn/state.py

```diff
@@ -9,7 +9,8 @@
 from tinygrad.features.multi import MultiLazyBuffer
 
 safe_dtypes = {"F16": dtypes.float16, "F32": dtypes.float32, "U8": dtypes.uint8, "I8": dtypes.int8, "I32": dtypes.int32, "I64": dtypes.int64,
-               "F64": dtypes.double, "B": dtypes.bool, "I16": dtypes.short, "U16": dtypes.ushort, "UI": dtypes.uint, "UL": dtypes.ulong}
+               "F64": dtypes.double, "BOOL": dtypes.bool, "I16": dtypes.short, "U16": dtypes.ushort, "UI": dtypes.uint, "UL": dtypes.ulong,
```

Comment:
I discovered that the key "B" in ```safe_dtypes``` is not correct during writing test test.
So, I fixed it as well.
And I also checked safetensors' definition is "BOOL".
https://github.com/huggingface/safetensors/blob/08db34094e9e59e2f9218f2df133b7b4aaff5a99/safetensors/src/tensor.rs#L639